File: Java/würfelRaten.java
import java.util.Scanner;

class würfelRaten {
    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {
            int value = ((int) (Math.random()*(7 - 1))) + 1;

            // Guess the number
            System.out.println("Guess a number betweeen 1 and 6!");
            int guess = scanner.nextInt();

            // Check if the guess is correct
            while (guess != value) {
                if (guess > 0 && guess < 7) System.out.println("Wrong! Try again!");
                if (guess < 1 || guess > 6) System.out.println("Systemoutofboundsexception");
                guess = scanner.nextInt();
            }

            // The Player won
            System.out.println("Correct!");
        }
    }
}

File: Python/würfelRaten.py
from random import random

value = int(random() * 6) + 1

print("Guess a number between 1 and 6!")

guess = int(input())

if guess > 0 and guess < 7:
    if guess == value:
        print("You guessed right!")
    else:
        print("You guessed wrong!")
        guess = int(input())

print("You won!")
